dune

```
@@ -0,0 +1,30 @@
+(env
+ (dev
+  (flags
+   (:standard -g (:include flags.sexp))))
+ (release
+  (flags
+   (:standard -g (:include flags.sexp)))))
+
+(executable
+ (name genconfig)
+ (modules genconfig)
+ (libraries dune.configurator))
+
+(rule
+ (targets cppo_flags)
+ (action
+  (run ./genconfig.exe)))
+
+(copy_files src/*.ml*)
+
+(library
+ (name extlib)
+ (public_name extlib_wrapped)
```

why this breakage ? if named extLib then user could still consume without noticing the switch to dune.
```suggestion
 (name extLib)
 (public_name extlib)
```